wireguard/rootfs/etc/cont-init.d/config.sh

```diff
@@ -291,6 +291,10 @@ for peer in $(bashio::config 'peers|keys'); do
     # Generate QR code with client configuration
     qrencode -t PNG -o "${config_dir}/qrcode.png" < "${config_dir}/client.conf"
 
+    # Show QR code with client configuration in logs
+    bashio::log.info "QR code for peer ${name}:"
+    qrencode -t ANSI < "${config_dir}/client.conf"
```

Comment:
The idea is nice, but when you have multiple clients, the logs won't do it (as it scrolls by and cannot be scrolled back).

Therefore this isn't a valid method IMHO.

Reply:
Totally agree. But I didn't want to throw an additionally settings parameter in there. Any good ideas on how we could resolve that?

1. Add an parameter `show_qrcode` as a **global** option which is true by default.
2. Add an parameter `show_qrcode` as a **peer** config option to show only the qr code of that peer in the logs which is by default false.
3. Only show the qrcode in the logs when the client is new or change, which I would imagine be not that user friendly.

Reply:
Ok fair point. Spaming the logs is not a good solution and there is currently no other way for extension to directly communicate such information. Maybe that is an idea for the future that could be added into HomeAssistant itself.